package.json

```diff
@@ -161,6 +164,7 @@
     "dateformat": "^3.0.3",
     "expo-font": "^8.1.1",
     "expo-step-indicator": "0.1.2",
+    "react-native-safe-area-context": "~3.0.7",
```

Comment:
`expo install react-native-safe-area-context` installs 3.1.7

Reply:
just reran the command in the example app and still got `~3.0.7`. Are you sure?

Reply:
You're right, thanks!